delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/Storages.scala

```diff
@@ -183,34 +182,6 @@ final class Storages private (
     } yield res
   }.span("updateStorage")
 
-  /**
-    * Add a tag to an existing storage
-    *
-    * @param id
-    *   the storage identifier to expand as the id of the storage
-    * @param projectRef
-    *   the project where the storage belongs
-    * @param tag
-    *   the tag name
-    * @param tagRev
-    *   the tag revision
-    * @param rev
-    *   the current revision of the storage
-    */
-  def tag(
```

Comment:
It should also not be possible to fetch by tag, we may have to introduce another type in `IdSegmentRef` that excludes the `Tag` one